website/content/v1.6/kubernetes-guides/configuration/local-storage.md

```diff
@@ -33,3 +33,51 @@ Both `EPHEMERAL` partition and user disks can be used for `hostPath` mounts.
 [Local Path Provisioner](https://github.com/rancher/local-path-provisioner) can be used to dynamically provision local storage.
 Make sure to update its configuration to use a path under `/var`, e.g. `/var/local-path-provisioner` as the root path for the local storage.
 (In Talos Linux default local path provisioner path `/opt/local-path-provisioner` is read-only).
+
+For example, Local Path Provisioner can be installed using [kustomize](https://kustomize.io/) with the following configuration:
+
+```yaml
+# kustomization.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- github.com/rancher/local-path-provisioner/deploy?ref=v0.0.26
+patches:
+- patch: |-
+    kind: ConfigMap
+    apiVersion: v1
+    metadata:
+      name: local-path-config
+      namespace: local-path-storage
+    data:
+      config.json: |-
+        {
+                "nodePathMap":[
+                {
+                        "node":"DEFAULT_PATH_FOR_NON_LISTED_NODES",
+                        "paths":["/var/local-path-provisioner"]
```

Comment:
We don't need to mount this into kubelet as well, right? It was required for hostPaths as I remember.

Reply:
it seems to work without it just fine, I haven't tested everything of course, but at least this gives a better start option for those poor souls like myself ;)